---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    entando.org/entando-resource-namespace: "my-namespace"
    entando.org/entando-resource-kind: "EntandoApp"
    EntandoApp: "my-app"
  name: "my-app-47687582"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "entando.org/v1"
    kind: "EntandoApp"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "7a470993-cfe7-4c2e-8f8d-c94c9d6afcb7"
action: "FAILED"
count: 1
firstTimestamp: "2021-07-14T06:18:48Z"
involvedObject:
  apiVersion: "entando.org/v1"
  kind: "EntandoApp"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "1509544"
  uid: "7a470993-cfe7-4c2e-8f8d-c94c9d6afcb7"
lastTimestamp: "2021-07-14T06:18:48Z"
message: "The deployment of EntandoApp my-namespace/my-app failed due to Fake Controller\
  \ Exception. Fix the root cause and then trigger a redeployment by adding the annotation\
  \ 'entando.org/processing-instruction: force'"
reason: "Failed"
related:
  apiVersion: "v1"
  kind: "Pod"
  name: "entando-k8s-entando-k8s-app-con-t4rhx-45-from-build-pack--p8kz7"
  namespace: "controller-namespace"
source:
  component: "entando-k8s-app-controller"
type: "Error"
